src/services/item/plugins/app/service.ts

```diff
@@ -61,8 +64,10 @@ export class AppService {
   async getContext(actorId, repositories: Repositories, itemId: string, requestDetails) {
     const { itemRepository, memberRepository } = repositories;
 
-    // TODO: check item is app
     const item = await itemRepository.get(itemId);
+    if (!isAppItem(item)) {
+      throw new Error('Item is not an app');
+    }
```

Comment:
Same as pointed above, this should be equivalent without changes to `item.type === ItemType.APP`. Is that the case?

Reply:
Checking the item.type like before does not narrow. This is why type-guards are required.